sub-packages/bionemo-moco/src/bionemo/moco/schedules/inference_time_schedules.py

```diff
@@ -458,3 +457,160 @@ def generate_schedule(
         if self.direction == TimeDirection.DIFFUSION:
             schedule = 1 - schedule
         return schedule
+    
+class EntropicInferenceSchedule(ContinuousInferenceSchedule):
+    """
+    Generates an entropic time schedule by remapping time based on the
+    cumulative information gain provided by a predictor function.
+    """
+    def __init__(
+        self,
+        predictor: Callable[[Tensor, Tensor], Tensor],
+        x_0_sampler: Callable[[int], Tensor],
+        x_1_sampler: Callable[[int], Tensor],
+        nsteps: int,
+        n_schedule_points: int = 100,
```

Comment:
nit: n_approx_entropy_points or soemthing just so its clear since nsteps and n schedule points sounds similar